includes/header-cleanup.php

```diff
@@ -0,0 +1,81 @@
+<?php
+/**
+ * Header cleanup.
+ *
+ * @package TenUpScaffold
+ */
+
+namespace TenUpScaffold\HeaderCleanup;
+
+/**
+ * Remove likely unused functionality that WordPress core enables
+ * by default.
+ *
+ * @return void
+ */
+function remove() {
+	// Remove the Emoji detection script.
+	// https://developer.wordpress.org/reference/functions/print_emoji_detection_script/
+	remove_action( 'wp_head', 'print_emoji_detection_script', 7 );
+	// Remove inline Emoji detection script.
+	// https://developer.wordpress.org/reference/functions/print_emoji_detection_script/
+	remove_action( 'admin_print_scripts', 'print_emoji_detection_script' );
+	// Remove Emoji-related styles from front end.
+	// https://developer.wordpress.org/reference/functions/print_emoji_styles/
+	remove_action( 'wp_print_styles', 'print_emoji_styles' );
+	// Remove Emoji-related styles from back end.
+	// https://developer.wordpress.org/reference/functions/print_emoji_styles/
```

Comment:
This comment can be combined with the same comment just above.

Reply:
fixed